go.mod

```diff
@@ -75,3 +75,5 @@ require (
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20180531100431-4c381bd170b4
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
+
+replace github.com/pingcap/parser => github.com/tangenta/parser v0.0.0-20190902053813-afcd61ce8608
```

Comment:
Before merge into mater branch, don't forget remove this line

Reply:
Ofcource.